tests/test_admin.py

```diff
@@ -2408,3 +2409,78 @@ def test_change_view_action_compare_versions_three_selected(self):
             response = self.client.post(endpoint, data, follow=True)
 
         self.assertContains(response, "Exactly two versions need to be selected.")
+
+
```

Comment:
We're still missing a lot of the tests for the admin actions. They will need to be ported across from navigation and pageadmin.